guides/intro-to-ploomber/_source.md

```diff
@@ -0,0 +1,106 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.6
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+
+# Your first Python pipeline
+
+<!-- start description -->
+To run this example locally, [install Ploomber](https://docs.ploomber.io/en/latest/get-started/quick-start.html) and execute: `ploomber examples -n guides/first-pipeline`
+
+Found an issue? [Let us know.](https://github.com/ploomber/projects/issues/new?title=guides/first-pipeline%20issue) Have questions? [Ask us anything on Slack.](https://ploomber.io/community/)
+<!-- end description -->
+
+# Ploomber Tutorial Intro
+
+- This should showcase the Ploomber value proposition. 
```

Comment:
I'd write this in terms of what they will do:

* run notebooks in parallel
* parametrize workflows
* generate reports

then say, we'll be using the covid dataset and put a link to it.